website/content/en/docs/helm/quickstart.md

```diff
@@ -313,4 +313,4 @@ For more information, refer [cli][addcli] doc.
 [helm-values]:https://helm.sh/docs/intro/using_helm/#customizing-the-chart-before-installing
 [helm-official]:https://helm.sh/docs/
 [addcli]: /docs/cli/operator-sdk_add_api
-[olm-user-guide]: /docs/olm-integration/user-guide
+[quickstart-bundle]: /docs/olm-integration/legacy/quickstart-bundle
```

Comment:
Ditto here: TODO to change this when helm KB plugin is merged.

/cc @camilamacedo86